docs/contributing/Style-guide-for-Flutter-repo.md

```diff
@@ -537,7 +537,7 @@ Example:
 abstract class Picture ...
 ```
 
-You can also use "See also" links, is in:
+You can also use "See also" links:
```

Comment:
This is a typo for "as in". The typo is good to fix.

But having an explicit lead-in is consistent with most of the other sections in the doc, which tend to say "Example:" or "For example:". So best to keep it, and just fix the typo.